validphys2/src/validphys/tests/test_regressions.py

```diff
@@ -127,6 +146,13 @@ def test_datasetchi2(data_singleexp_witht0_config):
     chi2s = API.groups_datasets_chi2_data(**data_singleexp_witht0_config)
     return results.fits_datasets_chi2_table(['test'], [exps], [chi2s])
 
+@make_table_comp(load_fits_chi2_table)
+def test_datasetchi2(data_singleexp_witht0_config):
+    # This is a bit hacky but avoids requiring a fit
+    exps = API.groups_data(**data_singleexp_witht0_config)
+    chi2s = API.groups_datasets_chi2_data(**data_singleexp_witht0_config)
+    return results.fits_datasets_chi2_table(['test'], [exps], [chi2s])
```

Comment:
I suppose we have things like `dataspecs_datasets_chi2_table` that could make this work in one go.

Reply:
These lines are actual a duplication, removed.